Documentation CYF 1.0/pages/api-functions-script.html

```diff
@@ -134,34 +145,6 @@ <h4>Examples</h4>
                 Encounter["torso"].Dust(true, true)
                 </pre>
                 </div>
-                <br>
```

Comment:
Why did you remove this part of the documentation? I think it's a good idea to know how to use script.Call() with several arguments